### ISSUE TITLE
No obvious way to add blocks on mobile

### ISSUE THREAD

**Issue by IFcoltransG** [nomeata/incredible#128]
Shift clicking and dragging is impossible with a touchscreen. Consider adding and advertising a different way to add blocks.

**Comment by nomeata** [comment]
Yeah, when I wrote the incredible proof machine, touch screens were still a bit less common as they are now.

For the UI I rely on JointJS, at a specific old version. Maybe upgrading to a new version would help, but I am doing a bit odd stuff (custom block shapes), so it won’t be trivial. I don't expect I’ll find time to do that any time soon, unfortunately.

**Comment by abliss** [comment]
The existing blocks on the playfield can be dragged around okay (android chrome  121.0.6167.18 ). I notice they have css class `ui-droppable` whereas the new blocks on the left, which cannot be touch-moved, have class `ui-draggable` . Maybe there's a much smaller change that can fix this without upgrading JointJS...
